templates/paragraph/paragraph.html.twig

```diff
@@ -32,8 +32,11 @@
  *   current user is a logged-in member.
  * - is_admin: Flag for admin user status. Will be true when the current user
  *   is an administrator.
+ * - parent_field: The field name that the paragraph is attached to.
+ * - parent_type: The entity type that the paragraph is attached to.
+ * - parent_bundle: The bundle of the entity that the paragraph is attached to.
  *
- * @see template_preprocess_paragraph()
+ * @see template_preprocess_paragraph() and gesso_preprocess_paragraph()
```

Comment:
I believe “and gesso_preprocess_paragraph()“ here is redundant since Drupal core’s use of `template_preprocess_paragraph()` here implies also looking at any module or theme overrides for this preprocess function.